### PR TITLE
Add/plugin panel shell

### PR DESCRIPTION
**Pull request by dmsnell**
See #3840 

Pulls in a visual shell of the new WordPress.com plugins page. This isn't yet hiding behind a feature flag and is accessible directly via the URL or by navigating to the plugins page for a Jetpack site and then switching to a WordPress.com site.

<a href="calypso.localhost:3000/plugins/wordbacon.wordpress.com">calypso.localhost:3000/plugins/wordbacon.wordpress.com</a>

screen shot 2016-03-21 at 4 19 27 pm

We can start to pick out which existing components we can use to build the UI and start creating some styling properties with this PR.

**Needed before merging**
 - [x] Hidden behind a feature-flag so we can prevent it from deploying to production
 - [ ] Can navigate to this spot when a WordPress.com site is selected in **My Sites**

**Testing**

Navigate to the plugins page on **My Sites**, you should see some plain text.

cc: @drw158 @roundhill

### CODE REVIEW

**Comment by artpi** [comment]
Dennis, this is awesome and I feel like we need it in Calypso, even with content from here: https://en.support.wordpress.com/plugins/ + simple explanation "**Y no plugins**".
I dont feel like we need to hide it behind feature flag or wait until the plugins page is ready.
Would be cool to have some nice visual, but I think even some basic info would be great.

The other possibility is - once people see "Plugins" in the menu, they'll be like "OMG! Plugins in WP.com ! Where do i install", without even reading the content
Still, I think that would GREATLY reduce number of support tickets and I'm a little jealous that I didnt think about this shell myself for post-support cleanup.

This would reduce confusion a lot!
@mtias @rralian @adambbecker  , what do you think?

File: package.json [hunk 1]
```diff
@@ -129,7 +129,7 @@
     "blanket": "1.1.6",
     "chai": "2.0.0",
     "deep-freeze": "0.0.1",
-    "enzyme": "1.1.0",
+    "enzyme": "2.2.0",
```
Comment (artpi) [inline package.json]: Could this change be in a different PR? :)

**Comment by artpi** [comment]
On my machine test go through. They are incredibly slow, but go through.
I ran master to also check it, but is also running incredibly slow..

File: package.json (hunk 1 above)
Comment (dmsnell) [inline package.json]: That's the plan, but this PR is more of a playground than production line. Thanks for the review!

**Comment by drw158** [comment]
@artpi That was the original issue in #3840, and we are still planning to do something similar to that, but it evolved a bit. Adding Plugins to the sidebar is a big change though, so we should wait until the page is ready (hopefully won't take too long, since it's static content). We also need to test the impact of adding the page.

**Comment by dmsnell** [comment]
Current unsettled visual:

screen shot 2016-03-22 at 6 08 05 pm

**Comment by dmsnell** [comment]
Reference: https://a8c.slack.com/files/davewhitley/F0TQD0W2C/Plugins_text

**Comment by gwwar** [comment]
:+1: Verified that I see stub data for wpcom sites on the plugins page. I also tested this with the following:
`DISABLE_FEATURES=manage/plugins/wpcom make run` and it correctly displayed the current `Oops! Not supported` notice.

Don't forget to squash before merging. :smile:
